traffic_ops/client/api_capability.go

```diff
@@ -0,0 +1,54 @@
+package client
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"github.com/apache/trafficcontrol/lib/go-tc"
+)
+
+/*
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+// GetAPICapabilities retrieves all (or filtered) api_capability from Traffic Ops.
```

Comment:
I think this ought to specify that an empty string in `capability` results in no filtering and tell callers what the `order` parameter does.